content/docs/0.5.0/reference/cli/index.md

```diff
@@ -261,15 +261,24 @@ keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAM
 
 ### keptn send event new-artifact
 
-After onboarding a service, the Keptn CLI allows pushing a new artifact for the service. This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images. The new artifact is pushed in the first stage specified in the `shipyard.yaml` file (usually this will be the dev stage). Afterwards, Keptn takes care of deploying this new artifact to the other stages.
+After onboarding a service, the Keptn CLI allows to trigger the deployment of an artifact.
+This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images. The new artifact is pushed in the first stage specified in the `shipyard.yaml` file (usually this will be the dev stage). Afterwards, Keptn takes care of deploying this new artifact to the other stages.
 
-To push a new artifact, use the command `send event new-artifact`, which sends a new-artifact-event to keptn in order to deploy a new artifact for the specified service in the provided project.
-Therefore, this command takes the project (`--project`), the service (`--service`), as well as the image (`--image`) and tag (`--tag`) of the new artifact.
+For notifying Keptn to deploy a new artifact for a service, use the command `send event new-artifact`.
+The CLI sends a new-artifact-event containing the image name and tag (not the image itself) to Keptn and then Keptn takes care
+of deploying this new artifact.
+This command takes the project (`--project`), the service (`--service`), as well as the image (`--image`) and tag (`--tag`) of the new artifact.
 
 ```console
 keptn send event new-artifact --project=PROJECTNAME --service=SERVICENAME --image=docker.io/keptnexamples/carts --tag=0.9.1
 ```
 
+**Note:** This command does not send the Docker image to Keptn. Instead, Keptn uses Kubernetes functionalities for pulling this image.
+Therefore, the used Docker registry has to accessible form your cluster. For pulling an image from a private registry,
+we would like to refer to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+Furthermore, please note that the value provided in the `image` flag has to contain your Docker registry. The only exception is `docker.io` because
```

Comment:
Please add an example here (e.g., ``--image=docker.io/mongo`` is the same as just stating ``--image=mongo``)

Reply:
Good point, thx